File: cases/taint/python/infeasible-branch-positive/infeasible_branch.py
def dfb_source():  # DFB-SOURCE: infeasible-branch-input
    return "tainted"


def dfb_sink(value):  # DFB-SINK: infeasible-branch-sink
    pass


def run():
    value = "clean"
    if True:
        value = dfb_source()  # DFB-WITNESS: feasible-tainted-branch
    dfb_sink(value)

File: cases/taint/ruby/infeasible-branch-positive/infeasible_branch.rb
def dfb_source # DFB-SOURCE: infeasible-branch-input
  "tainted"
end

def dfb_sink(value) # DFB-SINK: infeasible-branch-sink
end

def run
  value = "clean"
  if true
    value = dfb_source # DFB-WITNESS: feasible-tainted-branch
  end
  dfb_sink(value)
end
